### ISSUE TITLE
DownloadError: ERROR: Unable to extract uploader id

### ISSUE THREAD

**Issue by forceCpp** [ytdl-org/youtube-dl#31588]
hi i am using youtube-dl with discord.py it worked pretty good but now i am getting this error 

`Command raised an exception: DownloadError: ERROR: Unable to extract uploader id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.` 

```Traceback (most recent call last):
  File "/home/landrar/.local/lib/python3.10/site-packages/discord/ext/commands/core.py", line 229, in wrapped
    ret = await coro(*args, **kwargs)
  File "/home/landrar/devs/openDiscordBot/main.py", line 77, in play
    info = ytdl.extract_info(url, download=False)
  File "/home/landrar/.local/lib/python3.10/site-packages/youtube_dl/YoutubeDL.py", line 808, in extract_info
    return self.__extract_info(url, ie, download, extra_info, process)
  File "/home/landrar/.local/lib/python3.10/site-packages/youtube_dl/YoutubeDL.py", line 824, in wrapper
    self.report_error(compat_str(e), e.format_traceback())
  File "/home/landrar/.local/lib/python3.10/site-packages/youtube_dl/YoutubeDL.py", line 628, in report_error
    self.trouble(error_message, tb)
  File "/home/landrar/.local/lib/python3.10/site-packages/youtube_dl/YoutubeDL.py", line 598, in trouble
    raise DownloadError(message, exc_info)
youtube_dl.utils.DownloadError: ERROR: Unable to extract uploader id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/landrar/.local/lib/python3.10/site-packages/discord/ext/commands/bot.py", line 1349, in invoke
    await ctx.command.invoke(ctx)
  File "/home/landrar/.local/lib/python3.10/site-packages/discord/ext/commands/core.py", line 1023, in invoke
    await injected(*ctx.args, **ctx.kwargs)  # type: ignore
  File "/home/landrar/.local/lib/python3.10/site-packages/discord/ext/commands/core.py", line 238, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: DownloadError: ERROR: Unable to extract uploader id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.```


** updated youtube-dl and reinstalled it and changed the code, but i am still getting this error**

**Comment by MintKakapo** [comment]
Yeah, we're currently talking about this issue in my thread over at #31535 — I've listed some resources/other threads talking about what's happening. They're working on it!

**Comment by comicsansgreenki** [comment]
Looks like you didn't find #31530.

**Comment by MintKakapo** [comment]
> Looks like you didn't find #31530.

Whoops - - I'll reference that one from now on; I was leaving it open for discussion of coding developments—I was linking to mine for general discussion of the topic. Sorry about that

**Comment by dirkf** [comment]
And in general, #30839.

**Comment by DirtyDovi** [comment]
I haven't had any success using Youtube-DL [for at least a month or so] either.
I went through all of the notes, suggestions,  uninstalled everything / reinstalled everything . . . Nada.

Same 'error code': 

D:\VIDEO>youtube-dl.exe https://www.youtube.com/watch?v=B-hXUuz8ugg
[youtube] B-hXUuz8ugg: Downloading webpage
ERROR: Unable to extract uploader id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

And Everything is up to date.

No clue.
![Screenshot (489)](https://user-images.githubusercontent.com/127713582/224610077-0122c4bc-34b6-46f1-a0b2-2331ea6a2088.png)

**Comment by dimarhill** [comment]
[debug] Command-line config: ['-vU']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.03.04 [392389b7d] (pip)
[debug] Python 3.9.12 (CPython x86_64 64bit) - macOS-10.16-x86_64-i386-64bit (OpenSSL 1.1.1n  15 Mar 2022)
[debug] exe versions: none
[debug] Optional libraries: Cryptodome-3.17, brotli-None, certifi-2021.10.08, mutagen-1.46.0, sqlite3-2.6.0, websockets-10.4
[debug] Proxy map: {}
[debug] Loaded 1786 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.03.04, Current version: stable@2023.03.04
yt-dlp is up to date (stable@2023.03.04)

**Comment by dimarhill** [comment]
[debug] Command-line config: ['-vU']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.03.04 [392389b7d] (pip)
[debug] Python 3.9.12 (CPython x86_64 64bit) - macOS-10.16-x86_64-i386-64bit (OpenSSL 1.1.1n  15 Mar 2022)
[debug] exe versions: none
[debug] Optional libraries: Cryptodome-3.17, brotli-None, certifi-2021.10.08, mutagen-1.46.0, sqlite3-2.6.0, websockets-10.4
[debug] Proxy map: {}
[debug] Loaded 1786 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.03.04, Current version: stable@2023.03.04
yt-dlp is up to date (stable@2023.03.04)
